metals/src/main/scala/scala/meta/internal/metals/newScalaFile/NewFileTypes.scala

```diff
@@ -13,7 +13,7 @@ object NewFileTypes {
   case object ScalaFile extends NewFileType {
     override val id: String = "scala-file"
     override val syntax: Option[String] = None
-    override val label: String = "Scala file with automatically added package"
+    override val label: String = "Empty file"
```

Comment:
So this is more of a wdyt?

I personally fine the other one worded a bit oddly. When I first came across it I actually paused for a bit to make sure I understood what it meant. I honestly think we can just simplify this to `Empty file`. I think it gets the point across and does what most people would expect, just adding in the package. wdyt?